backend/app/views/reports/top_level_subreport.erb

```diff
@@ -3,6 +3,7 @@
 <% sub_report_data.each do |item| %>
   <div class="sub-report-item">
     <% item.each do |key, value| %>
+      <% next if value && value.empty? %>
```

Comment:
```suggestion
      <% next if value.respond_to(:empty?) && value.empty? %>
```

Reply:
@mark-cooper I just added this because I found the report I tested failed due to a value in the `Time` class showing up here. I don't know enough to offer a better solution, but this at least keeps things going.